packages/nest/test/open-feature-sdk.spec.ts

```diff
@@ -160,4 +161,16 @@ describe('OpenFeature SDK', () => {
       expect(evaluationSpy).toHaveBeenCalledWith('testBooleanFlag', false, { targetingKey: '123' }, {});
     });
   });
+
+  describe('evaluation context service should', () => {
+    it('inject the evaluation context from contex factory', async function () {
+      const evaluationSpy = jest.spyOn(defaultProvider, 'resolveBooleanEvaluation');
+      await supertest(app.getHttpServer())
+        .get('/dynamic-context-in-service')
+        .set('x-user-id', 'dynamic-user')
```

Comment:
I see x-user-id being used multiple times.
We would probably benefit from sketching conventions about OF context propagation between services to make flag evaluation consistent and this SDK might be a good starting point for this.
Marking @toddbaert for visibility

Reply:
What exactly do you mean by this?

Reply:
If we have a convention for open feature context building and passing in requests we can propagate it across multiple services to make flag resolution consistent without much effort. 
Eg: Considering that service A (Node) has access to userId/tragetingKey, it can then forward this info as request metadata to services B (Golang), C (Java).
A -> B
   -> C
   
 I don´t  think I made myself clear enough though. Did you get the idea?

Reply:
Baggage could be a cool solution for context propagation.
To me, this would be a special version of a `contextFactory`, that we could provide later if we spec the usage of something like baggage I would say. I feel like this is too early to include into this SDK.
What do you think @luizgribeiro?

Reply:
> I feel like this is too early to include into this SDK.
I agree.

I'd say that since we are dealing with a context factory it will probably be affected in the future is baggage is included in OF. It would be necessary to provide a way to read it and insert in current context (may a higher level contextFactory that also runs the user provided context factory :thinking:).

It would also be necessary to use the provided baggage in different interfaces with other services (HTTP, gRPC, etc). I think we might learn a lot from how Otel is handling this, but it's out of scope from this SDK perspective.